test/unit/uk/gov/hmrc/tariffclassificationfrontend/controllers/CasesControllerSpec.scala

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.tariffclassificationfrontend.controllers
+
+import org.mockito.BDDMockito._
+import org.scalatest.mockito.MockitoSugar
+import org.scalatest.{Matchers, WordSpec}
+import org.scalatestplus.play.guice.GuiceOneAppPerSuite
+import play.api.http.Status
+import play.api.i18n.{DefaultLangs, DefaultMessagesApi}
+import play.api.test.FakeRequest
+import play.api.test.Helpers._
+import play.api.{Configuration, Environment}
+import uk.gov.hmrc.tariffclassificationfrontend.config.AppConfig
+import uk.gov.hmrc.tariffclassificationfrontend.service.CasesService
+
+import scala.concurrent.Future
+
+
+class CasesControllerSpec extends WordSpec with Matchers with GuiceOneAppPerSuite with MockitoSugar {
+  private val fakeRequest = FakeRequest("GET", "/")
+
+  private val env = Environment.simple()
+  private val configuration = Configuration.load(env)
+
+  private val messageApi = new DefaultMessagesApi(env, configuration, new DefaultLangs(configuration))
+  private val appConfig = new AppConfig(configuration, env)
+  private val service = mock[CasesService]
+
+  private val controller = new CasesController(service, messageApi, appConfig)
+
+  "GET /cases" should {
```

Comment:
We are not testing the route here, but only `controller.gateway()`.
So, you can change the title to simply "gateway()"

Reply:
Done